src/objc_stubs.c

```diff
@@ -10,11 +10,12 @@
  */
 
 #include "arraytypes.h"
-#include "class.c"
+#include "aggregate.h"
 #include "mars.h"
 #include "objc.h"
```

Comment:
You should move the `#include "objc.h"` up to the top of the include list in `objc.c`, to prevent this from happening.